### PR TITLE
Liveblog epic test round 1b: no italics

### PR DESCRIPTION
**Pull request by tomrf1**
In the previous liveblog epic design test there was no significant difference against the control.

The next round is to try without italics.

### Control
Screen Shot 2020-10-27 at 14 34 10

### Variant
Screen Shot 2020-10-27 at 14 32 44

### Copy for US
Screen Shot 2020-10-27 at 14 33 13

### Copy for non-US, on US politics tags
Screen Shot 2020-10-27 at 14 32 54

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/experiments/tests/liveblog-epic-design-test.js
Comment (TomPretty) [inline static/src/javascripts/projects/common/modules/experiments/tests/liveblog-epic-design-test.js:24]: the screenshot looks fine, but I think there's a double space here?

Comment (tomrf1) [reply static/src/javascripts/projects/common/modules/experiments/tests/liveblog-epic-design-test.js:24]: good spot!

**Review by tompretty — APPROVED** [review]
LGTM 👍 will be interesting to see if looking more like regular liveblog blocks improves the conversions!

**Review by mxdvl — APPROVED** [review]

**Comment by paperboyo** [comment]
Italics makes it stand out as separate from other blocks, this is important. Lengthy italics read worse, so this may be a successful test, but it’s flawed because it removes the visual distinction. I’m sure we do not want readers feel tricked into reading this, so in my humble opinion we need a way to distinguish it from the rest of the content (just as we do with epic – it doesn’t look like an article).

**Comment by tomrf1** [comment]
@paperboyo this is a great point (and one I've raised!). I'm told we have Editorial approval for this test, but feel free to message the Contributions team about it

**Pull request merged**
